test/team_test.rb

```diff
@@ -0,0 +1,136 @@
+require "pry"
+require "minitest/autorun"
+require "minitest/pride"
+require "./lib/team"
+require "./lib/player"
+
+class TeamTest < Minitest::Test
+
+  def test_if_it_exists
+    france = Team.new("France")
+
+    assert_instance_of Team, france
+  end
+
+  def test_the_country_name
+    france = Team.new("France")
+
+    assert_equal "France", france.country
+  end
+
+  def test_if_player_array_is_empty
+    france = Team.new("France")
+
+    assert_equal [], france.players
+  end
+
+  def test_if_team_is_eliminated
+    france = Team.new("France")
+
+    assert_equal false, france.eliminated?
+  end
+
+  def test_if_team_is_eliminated_equals_true
+    france = Team.new("France")
+
+    assert_equal true, france.eliminated
+  end
+
+  def test_if_team_is_eliminated_equals_true_again
+    skip
+    france = Team.new("France")
+
+    assert_equal true, france.eliminated?
+  end
+
+  def test_if_it_adds_new_player
```

Comment:
This test looks good, but it is a bit of overkill!  You have already tested the player can be created in the player test and you are testing that the player is added to the team in the test called `test_if_it_adds_player_to_array`